CHANGES.rst

```diff
@@ -18,7 +18,10 @@ New features:
 
 Bug fixes:
 
-- ...
+- vBoolean can now be used as an parameter
```

Comment:
I think the fix covers all the property types...  `vInt`, `vTime`, etc

Reply:
You're probably right, but I don't put things in the changelog if there isn't a corresponding test for the entry.

Reply:
If you put it in the changelog, it creates an expectation and then people come back if this is not the case. But since it is in the documentation, the expectation is there either way.